blocks/extensions/training.py

```diff
@@ -0,0 +1,45 @@
+import inspect
+from blocks.extensions import SimpleExtension
+
+
+class SharedVariableModifier(SimpleExtension):
+    """Adjusts shared variable parameter using some function.
+
+    Applies function to compute the new value of a shared parameter each
+    iteration.
+
+    This class can be used to adapt over the training process parameters
+    like learning rate, momentum, etc.
+
+    Parameters
+    ----------
+    parameter : :class:`~tensor.TensorSharedVariable`
+        shared variable to be adjusted
```

Comment:
Sorry for being picky, but we everywhere else in Blocks the parameter docstrings are started with a capital letter.